src/Features/CSharp/Portable/GenerateMember/GenerateParameterizedMember/CSharpGenerateParameterizedMemberService.cs

```diff
@@ -112,7 +112,7 @@ protected override ImmutableArray<ITypeParameterSymbol> GenerateTypeParameters(C
                 CancellationToken cancellationToken)
             {
                 var methodTypeParameter = GetMethodTypeParameter(type, cancellationToken);
-                return methodTypeParameter ?? CodeGenerationSymbolFactory.CreateTypeParameterSymbol(NameGenerator.GenerateUniqueName("T", isUnique)).WithNullability(NullableAnnotation.None);
+                return methodTypeParameter ?? (ITypeParameterSymbol)CodeGenerationSymbolFactory.CreateTypeParameterSymbol(NameGenerator.GenerateUniqueName("T", isUnique)).WithNullableAnnotation(NullableAnnotation.None);
```

Comment:
❔ Why do we need to call `WithNullableAnnotation` here?

Reply:
Good catch.

Reply:
(previously this would have triggered the wrap which could have changed equality comparison behavior, but not anymore)